src/main/java/racingcar/model/Car.java

```diff
@@ -0,0 +1,49 @@
+package racingcar.model;
+
+import camp.nextstep.edu.missionutils.Randoms;
+
+public class Car {
+	private static final int MAX_NAME_LENGTH = 5;
+    private static final int MIN_NAME_LENGTH = 1;
+    private static final String NAME_FORMAT_REGEX = "^[a-zA-Z0-9]*$";
+    private final String name;
```

Comment:
```suggestion
	private static final int MAX_NAME_LENGTH = 5;
    private static final int MIN_NAME_LENGTH = 1;
    private static final String NAME_FORMAT_REGEX = "^[a-zA-Z0-9]*$";
    
    private final String name;
```

1주차 공통 피드백에 이런 내용이 있었네요.
> ### 공백 라인을 의미 있게 사용한다
> 공백 라인을 의미 있게 사용하는 것이 좋아 보이며, 문맥을 분리하는 부분에 사용하는 것이 좋다. 과도한 공백은 다른 개발자에게 의문을 줄 수 있다.

상수와 필드는 구분을 해 보이는 것도 가독성에 도움을 줄 수 있을 것 같네요.